### DISCUSSION TITLE
As editor in a site with many variants, I'd like to publish all the languages at once, without clicking on all the checkboxes

### DISCUSSION THREAD

**Discussion by simonech** [umbraco/Umbraco-CMS#15297] [Features and ideas]
Similar concert to https://github.com/umbraco/Umbraco-CMS/discussions/15296:
when I have 24 languages it's very annoying having to select 24 checkboxes to be able to publish all the variants at once.

![image](https://github.com/umbraco/Umbraco-CMS/assets/61557/ca843798-d537-4f61-a14b-20ed111d81b0)

It would be great if there was a "select all" checkbox.

**Comment by lassefredslund** [comment]
Hi @simonech
I think this would make a lot of sense. I've set this discussion to "up for grabs".

**Comment by simonech** [comment]
PR sent: https://github.com/umbraco/Umbraco-CMS/pull/15776

**Comment by simonech** [comment]
This feature was released in 13.3.0. I'll close the issue
